pillar/heroku/bootcamps.sls

```diff
@@ -0,0 +1,119 @@
+{% set minion_id = salt.grains.get('id', '') %}
+{% set environment = minion_id.split('-')[-1] %}
+{% set rds_endpoint = salt.boto_rds.get_endpoint('bootcamps-rds-postgresql') %}
+
+{% set env_dict = {
+    'ci': {
+      'app_name': 'xpro-ci',
+      'env_name': 'ci',
+      'BOOTCAMP_ADMISSION_BASE_URL': 'http://admissions-test.herokuapp.com',
+      'BOOTCAMP_ECOMMERCE_BASE_URL': 'https://bootcamp-ecommerce-ci.herokuapp.com',
+      'BOOTCAMP_LOG_LEVEL': 'INFO',
+      'BOOTCAMP_SUPPORT_EMAIL': 'bootcamp-support@mit.edu',
+      'CYBERSOURCE_REFERENCE_PREFIX': 'ci',
+      'EDXORG_BASE_URL': 'https://micromasters.d.mitx.mit.edu',
+      'GA_TRACKING_ID': 'UA-5145472-19',
+      'GTM_TRACKING_ID': 'GTM-NZT8SRC',
+      'vault_env_path': 'rc-apps'
+      },
+    'rc': {
+      'app_name': 'xpro-rc',
+      'env_name': 'rc',
+      'BOOTCAMP_ADMISSION_BASE_URL': 'http://admissions-test.herokuapp.com',
+      'BOOTCAMP_ECOMMERCE_BASE_URL': 'http://bootcamp-rc.odl.mit.edu/',
+      'BOOTCAMP_LOG_LEVEL': 'INFO',
+      'BOOTCAMP_SUPPORT_EMAIL': 'bootcamp-support@mit.edu',
+      'CYBERSOURCE_REFERENCE_PREFIX': 'rc',
+      'EDXORG_BASE_URL': 'https://courses.stage.edx.org',
+      'GA_TRACKING_ID': 'UA-5145472-19',
+      'GTM_TRACKING_ID': 'GTM-NZT8SRC',
+      'vault_env_path': 'rc-apps'
+      },
+    'production': {
+      'app_name': 'xpro-production',
+      'env_name': 'production',
+      'BOOTCAMP_ADMISSION_BASE_URL': 'http://admissions.herokuapp.com',
+      'BOOTCAMP_ECOMMERCE_BASE_URL': 'https://bootcamp.odl.mit.edu/',
+      'BOOTCAMP_LOG_LEVEL': 'INFO',
+      'BOOTCAMP_SUPPORT_EMAIL': 'bootcamp@mit.edu',
+      'CYBERSOURCE_REFERENCE_PREFIX': 'prod',
+      'EDXORG_BASE_URL': 'https://courses.edx.org',
+      'GA_TRACKING_ID': 'UA-5145472-18',
+      'GTM_TRACKING_ID': 'GTM-NZT8SRC',
+      'vault_env_path': 'production-apps'
+      }
+} %}
+{% set env_data = env_dict[environment] %}
+{% set business_unit = 'bootcamps' %}
+{% set cybersource_creds = salt.vault.read('secret-' ~ business_unit ~ '/' ~ env_data.vault_env_path ~ '/cybersource') %}
+{% set fluidreview = salt.vault.read('secret-' ~ business_unit ~ '/' ~ env_data.vault_env_path ~ '/fluidreview') %}
```

Comment:
Need to append `.data` to the `vault.read` call so that you don't have to include `fluidreview.data.foobar` in all of the Jinja references below.